tests/test_graphviz_integration.py

```
@@ -0,0 +1,116 @@
+from typing import cast
+from unittest.mock import MagicMock, patch
+
+from temoa.core.config import TemoaConfig
+from temoa.model_checking.commodity_graph import generate_graphviz_files, visualize_graph
+from temoa.model_checking.network_model_data import NetworkModelData
+from temoa.types.core_types import Commodity, Period, Region, Sector, Technology, Vintage
+from temoa.utilities.graph_utils import EdgeTuple
+
+
+class TestGraphvizIntegration(unittest.TestCase):
+    def setUp(self) -> None:  # noqa: N802
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `N802`)

Remove unused `noqa` directive

(RUF100)